jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
+    def __init__(self):
+        self.player_list = []
+        self.current_player = 0
+        self.spin_counter = 50
+        self.current_round = 1
+        self.turnPoints = 0
+        self.question_bank
+
+    'Add a player to the list'
+    def addPlayer(self, name):
+        Player = player(name)
+        self.player_list.append(Player)
+        return
+
+    'Remove a player from the list'
+    def deletePlayer(self, player):
+        self.player_list.remove(player)
```

Comment:
Probably want to add some debugging here. We will want to enumerate the list of players after we delete one to show that the function is working properly.

Reply:
I've added a print statement to declare we're deleting a player. I see we have a logger, should I make an instance of that instead, or is there a special way Python performs logging?